PalasoUIWindowsForms/ImageToolbox/ImageToolboxControl.cs

```diff
@@ -218,11 +214,9 @@ private void listView1_SelectedIndexChanged(object sender, EventArgs e)
 
 				if (_currentControl != null)
 				{
-					GetImageFromCurrentControl();
-
 					_panelForControls.Controls.Remove(_currentControl);
 					((IImageToolboxControl) _currentControl).ImageChanged -= new EventHandler(imageToolboxControl_ImageChanged);
-					_currentControl.Dispose();
```

Comment:
Looking further, AddControl is private and is used only in OnLoad, being passed a function which creates an AcquireImageControl. There seems no reason for this complexity. If we always make one of those when an image is selected, why not just let the select code know it directly? Much more readable, and it would be unambiguous that we ought to dispose of _currentControl if it isn't null.

That leaves the question, why is it a problem to dispose the control, if indeed that was causing problems? Maybe the AcquireImageControl is disposing something it shouldn't?